src/app/design/frontend/base/default/template/banana/footer.phtml

```diff
@@ -0,0 +1,15 @@
+<!-- // BEGIN // Banana Stand Integration code. See https://www.fera.ai for more information. -->
+    <script>
+      window.__bsio = window.__bsio || []; window.__bsioDebugMode = true;
```

Comment:
You can check if devMode is on in Magento, and if so you can include this. Else, don't include it.

Reply:
Ah I missed cleaning that up, good catch!